Playbooks/Enrich-MalwareBazaar/Playbook/azuredeploy.json

```diff
@@ -0,0 +1,454 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "metadata": {
+        "title": "Enrich file hashes entities - MalwareBazaar",
+        "description": "This playbook will enrich a Sentinel Incident with hash information from MalwareBazaar.",
+        "mainSteps": [
+            "Only file hashes entity types will be enriched with this playbook"
+        ],
+        "prerequisites": [
+            "1. Logic Apps Custom Connector MalwareBazaar.\n2.Logic App managed identity should be given Sentinel Responder role to read incident trigger and write comment/tag to incident."
+        ],
+        "postDeployment": [
+            "None"
+        ],
+        "prerequisitesDeployTemplateFile": "../CustomConnector/azuredeploy.json",
+        "lastUpdateTime": "2023-01-08T10:00:00.000Z",
+        "entities": [
+            "FileHash"
+        ],
+        "tags": [
+            "Enrichment",
+            "MalwareBazaar"
+        ],
+        "support": {
+            "tier": "community"
+        },
+        "author": {
+            "name": "juju4"
+        },
+        "source": {
+            "kind": "Community"
+        },
+        "version": "1.0.0",
+        "releaseNotes": [
+            {
+                "version": "1.0.0",
+                "title": "Enrich file hashes entities - MalwareBazaar",
+                "notes": [
+                    "Initial version"
+                ]
+            }
+        ]
+    },
+    "parameters": {
+        "resourceTags": {
+            "type": "object",
+            "defaultValue": {
+                "LogicAppsCategory": "security"
+            }
+        },
+        "PlaybookName": {
+            "defaultValue": "Enrich-MalwareBazaar",
+            "type": "string"
+        },
+        "customApis_MalwareBazaar_name": {
```

Comment:
can we mention this as "custom connector" ? instead of "customApis_MalwareBazaar_name"